srcdeps-core/src/main/java/org/l2x6/srcdeps/core/config/ScmRepository.java

```diff
@@ -157,13 +223,29 @@ public boolean equals(Object obj) {
     }
 
     /**
-     * @return an identifier of this {@link ScmRepository}. Only {@code [a-zA-Z0-9_]} characters are allowed.
+     * @return an identifier of this {@link ScmRepository}. Should be a sequence of Java identifiers concatenated by
+     *         {@code '.'} character
      */
     public String getId() {
         return id;
     }
 
     /**
+     * @return a {@link Path} created out of {@code '.'} delimited segments of {@link #id}. If {@link #id} is
+     *         {@code "org.project.component"} then the {@link Path} returned by this method will be
+     *         {@code "org/project/component"}
+     */
+    public Path getIdAsPath() {
+        List<String> pathElements = new ArrayList<>();
+        StringTokenizer st = new StringTokenizer(id, String.valueOf(ID_DELIMITER));
```

Comment:
> Unless there is a certain performance concern

A performance concern is always there :) plus I need the `first` separated from the rest of the segments (because the `Path.get()` wants it like that) - I'd have to copy the id.split(ID_DELIMITER) array starting at element 1. Thus -1